frame/support/procedural/src/lib.rs

```diff
@@ -1004,6 +1004,16 @@ pub fn no_default(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
+#[proc_macro_attribute]
+pub fn no_bounds(_: TokenStream, _: TokenStream) -> TokenStream {
+	pallet_macro_stub()
+}
+
+#[proc_macro_attribute]
+pub fn verbatim(_: TokenStream, tokens: TokenStream) -> TokenStream {
+	tokens
```

Comment:
verbatim should also be a `pallet_macro_stub()`

Reply:
Since `register_default_impl` also exports tokens, marking this as `pallet_macro_stub()` was causing a compiler error. It's only at the later time when we use `derive_impl` that this macro is correctly parsed.

Reply:
ahh ok